src/gui/components/wxTitleManagerList.cpp

```diff
@@ -200,21 +200,21 @@ boost::optional<wxTitleManagerList::TitleEntry&> wxTitleManagerList::GetTitleEnt
 
 boost::optional<const wxTitleManagerList::TitleEntry&> wxTitleManagerList::GetTitleEntry(const fs::path& path) const
 {
-	const auto tmp = path.generic_u8string();
+	const auto tmp = path.generic_string();
```

Comment:
This changes what encoding you get on Windows from UTF-8 to the system codepage, which I'm assuming is not intended. Would this work as an alternative?

```suggestion
	const auto tmp = _utf8Wrapper(path);
```